packages/api/test/e2e/promise-queries.spec.ts

```diff
@@ -9,7 +9,7 @@ import Api from '../../src/promise';
 const keyring = testingPairs();
 
 describe.skip('e2e queries', () => {
-  let api;
+  let api: any;
```

Comment:
Unless we are constructing a specific mocked API (e.g. see https://github.com/polkadot-js/api/blob/master/packages/api/src/promise/SubmittableExtrinsic.spec.ts#L17), this should not be any. I believe it sort-of defats the purpose just renaming with a blanket instead of using TS.